Feat/add reversal android

JoshM1994 opened the pull request:
* Adds the `reverse` and `boomerang` abilities to Android

* iOS equivalents to follow later today

shahen94 commented on lib/ProcessingManager/ProcessingManager.android.js:
```diff
@@ -49,6 +49,16 @@ export class ProcessingManager {
     return TrimmerManager.compress(actualSource, options);
   }
 
+  static boomerang(source: sourceType, options: any): Promise<*> {
```
Can you please specify options type?

shahen94 commented on lib/ProcessingManager/ProcessingManager.android.js:
```diff
@@ -49,6 +49,16 @@ export class ProcessingManager {
+    return TrimmerManager.boomerang(actualSource, options);
+  }
+
+  static reverse(source: sourceType, options: any): Promise<*> {
```
Here as well

JoshM1994 commented:
Thanks for the feedback @shahen94 

- Added example usage to the README
- Removed the options entirely since they are not used
- Modified the return promise to be consistent with `trim` and `crop`

shahen94 commented:
@JoshM1994 You're awesome. Thanks !
